staging/volumes/storageos/README.md

```diff
@@ -0,0 +1 @@
+This file has moved to [https://github.com/kubernetes/examples/blob/master/staging/volumes/storageos/README.md](https://github.com/kubernetes/examples/blob/master/staging/volumes/storageos/README.md)
```

Comment:
Isn't this file pointing to itself? It might be good to dig up the actual file (which is supposed to be here already).